src/test/java/com/company/spring_application/connector/ConnectorCSVTest.java

```diff
@@ -0,0 +1,47 @@
+package com.company.spring_application.connector;
+
+import com.company.spring_application.domain.Order;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+
+public class ConnectorCSVTest {
```

Comment:
1. No need to setup spring context in this test.
2. ConnectorCSV should be tested only via public methods.
3. Read about unit test mocks and assertions.
4. Test should assert that properly build Order is passed to Connector.